src/FlashAlert/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from '../../util/withStyles';
+import styles from './index.scss';
+
+class FlashAlert extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      shown: true
+    };
```

Comment:
I don't think you need to have this state in here. Remember, the parent can handle that :) you should just make sure this component accepts a function as a prop called `onDismiss` and let the parent of the component decide to hide/show!